### ISSUE TITLE
RStudio interface: dependency problems

### ISSUE THREAD

**Issue by pablobernabeu** [jupyterhub/binderhub#948]
Hello,

I would like to ask regarding the *jovyan* RStudio interface for Binder. This interface may enable something quite unique right now, unmatched by alternatives, so thank you for it.

I would like to ask whether it would be remotely possible to keep the underlying R and RStudio software up to date. The reason is that I've found package dependency problems (for this repo: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/pablobernabeu/Modality-exclusivity-norms-747-Dutch-English-replication/master?urlpath=rstudio)) such as

`ERROR: dependency ‘mvtnorm’ is not available for package ‘multcomp’`

Thank you very much

**Comment by betatim** [comment]
Hello Pablo 👋

Thanks for the kind words! There is [work happening](https://github.com/jupyter/repo2docker/pull/772) to make the version of R configurable for each repository. Do you know if being able to choose a different version of R would solve your problem?

Independent of that we should think about updating the version of RStudio.

**Comment by pablobernabeu** [comment]
Hi Tim :wave:

Thank you. That sounds great. Certainly, being able to choose a different version would solve my problem. I've just found a similar error, explicitly pointing to the version:

```
> install.packages('car')
Installing package into ‘/srv/rlibs’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘car’ is not available (for R version 3.4.4)
```

I've realised the package issue is specifically about R, so RStudio is a less pressing issue for me. Still, if it can be updated too, I guess not bad at all!

Thank you and all the best
